packages/substrate/src/__test__/fungible.test.ts

```diff
@@ -16,48 +16,15 @@ import {
 } from '../utils/index.js';
 
 jest.mock('../utils/index.js');
-jest.mock('@polkadot/api', () => {
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-  const originalModule = jest.requireActual('@polkadot/api');
-  const mockBalance = {
-    data: {
-      free: '500',
-      reserved: '5000',
-      miscFrozen: '5000',
-      feeFrozen: '5000',
-    },
-  };
-  const mockChainProperties = {
-    tokenDecimals: ['12'],
-    tokenSymbol: ['DOT'],
-  };
-
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-  return {
-    ...originalModule,
-    WsProvider: jest.fn().mockImplementation(() => ({})),
-    ApiPromise: {
-      create: jest.fn().mockResolvedValue({
-        query: {
-          system: {
-            account: jest.fn().mockResolvedValue(mockBalance),
-          },
-        },
-        registry: {
-          getChainProperties: jest.fn().mockReturnValue(mockChainProperties),
-        },
-      }),
-    },
-  };
-});
+
+const RHALA_RPC_URL = 'wss://rhala-node.phala.network/ws';
+const wsProvider = new WsProvider(RHALA_RPC_URL);
```

Comment:
should the provider be mocked?

Reply:
it might initiate a lot of calls unintentionally, we should mock the provider

Reply:
It's already mocked globally in test/setupJest.js  I removed it from fungible test to avoid confusions

Reply:
oh so this is a mocked instance instead of an actual instance

Reply:
@saadjhk yeah exactly